edi_webservice/models/edi_backend.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2020 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class EdiBackend(models.Model):
+
+    _inherit = "edi.backend"
+
+    webservice_backend_id = fields.Many2one("webservice.backend")
+    _webservice_actions = ("send", "receive")
+
+    def _get_component_usage_candidates(self, exchange_record, key):
+        candidates = super()._get_component_usage_candidates(exchange_record, key)
+        if not self.webservice_backend_id or key not in self._webservice_actions:
+            return candidates
+        return ["webservice.{}".format(key)] + candidates
+
+    def _component_match_attrs(self, exchange_record, key):
+        # Override to inject storage_backend_type
```

Comment:
```suggestion
        # Override to inject `webservice_protocol` as match attribute
```
and replace all occurrences :wink: